src/v2/Apps/FeatureAKG/__tests__/FeatureAKGRoute.jest.tsx

```diff
@@ -79,6 +79,7 @@ describe("FeatureAKG", () => {
         trackEvent,
       }
     })
+    jest.setTimeout(10000)
```

Comment:
This test became flaky and timed out a fewer times, I increased the timeout of it but in the future would be better to refactor it, but more than 1k lines in a single test file and I have no context about this AKG guy